cmd/common.go

```
@@ -171,6 +172,20 @@ func ReadValuesFiles() error {
 	return nil
 }
 
+// Copied from https://github.com/helm/helm/blob/master/pkg/engine/funcs.go#L83
+// toYAML takes an interface, marshals it to yaml, and returns a string. It will
+// always return a string, even on marshal error (empty string).
+//
+// This is designed to be called from a template.
+func toYAML(v interface{}) string {
+	data, err := yaml.Marshal(v)
+	if err != nil {
```

Why wouldn't we want to throw the error here?

---

Just followed the link to the helm version of `toYAML` disregard